src/ontology/edits/oeo-physical.omn

```diff
@@ -814,6 +814,9 @@ Class: <http://purl.obolibrary.org/obo/BFO_0000023>
     
 Class: <http://purl.obolibrary.org/obo/BFO_0000024>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A tangential proper part is a fiat object part that conforms to the qualitative spatially representaion of TPP in RCC8."@de
```

Comment:
Here is again the unwanted addition of a definition

Reply:
Sorry, my description was not precise.
In case there are additional commits in the branch that I merged initially, it has to be merged again into this to catch up the changes.

I did the merge, so everything should have caught up now.
Hope that's allright now ;)